zq2/docs/endpoints.md

```diff
@@ -1,29 +1,41 @@
 ---
 id: endpoints
-title: API endpoints
-hide:
-  - toc
+title: Endpoints, Block Explorer and Faucet
 keywords:
   - api
   - introduction
   - endpoints
-description: Zilliqa API Endpoints
+description: Endpoints, Block Explorer and Faucet
 ---
 
-# Endpoints
+# Endpoints, Block Explorer and Faucet
 
-Chain ids can be found at [chainlist](https://chainlist.org/?search=zilliqa&testnets=true).
+As we introduce additional test networks for Zilliqa 2.0, their public RPC node URLs and chain IDs will be published on [ChainList](https://chainlist.org/?search=zilliqa&testnets=true) alongside existing chain information for Zilliqa 1.0.
 
-Uptime can be found at [status.zilliqa.com](https://status.zilliqa.com) and you can get API examples from the [api](/api) pages.
+## Networks
+Details for the current network versions of Zilliqa 2.0 are listed below.
 
-We have several blockchains running.
+<div class="table" markdown>
+|  Name  | Version | ChainId | API | Otterscan | Faucet  |
+| ------ | ------- | ------- | --- | --------- | ------  |
+| proto-testnet | xx | xx | xx | xx | xx | xx | xx |
+</div>
 
-## devnet
+*__Note__ that this is currently a proto-testnet version of Zilliqa 2.0 designed primarily for testing purposes, and therefore its state and uptime are not guaranteed*
 
-Devnet is a general purpose developer network, synchronised to the latest (or nearly the latest) commit from [https://github.com/zilliqa/zq2](https://github.com/zilliqa/zq2) . Its state and uptime are not guaranteed, but it is useful for checking your contracts against the latest and greatest software versions.
+## Block Explorer
 
-<div class="center-table" markdown>
-|  Name  | Version | ChainId | API | Otterscan | Faucet  | Connect | Notes |
-| ------ | ------- | ------- | --- | --------- | ------  | ------- | ----- |
-| devnet | <span id="devnet_vsn" class="zq2_vsn">api.zq2-devnet.zilliqa.com</span> | [33469](https://chainlist.org/chain/33469) | https://api.zq2-devnet.zilliqa.com | https://explorer.zq2-devnet.zilliqa.com | https://faucet.zq2-devnet.zilliqa.com/ | <a href="javascript:connectZilliqaChain('https://api.zq2-devnet.zilliqa.com', '0x82BD', 'Zilliqa 2 EVM Devnet', 'https://explorer.zq2-devnet.zilliqa.com', 'Zilliqa 2 testnet', 'ZIL')">Metamask</a> | <span class="zq2_docs_devnet_vsn" kind="api">&nbsp;</span> |
```

Comment:
Could we please keep, if not this table, then the <span's> ? They provide the links developers need to interact with the testnet, and automatically query the version currently running.

Reply:
https://github.com/Zilliqa/dev-portal/issues/23